action/rlp_tx_test.go

```diff
@@ -90,374 +90,209 @@ func TestGenerateRlp(t *testing.T) {
-	chainID  uint32
-	encoding iotextypes.Encoding
-	data     []byte
-	hash     string
-	pubkey   string
-	pkhash   string
+	actType       string
+	nonce, limit  uint64
+	price, amount string
+	to            string
+	chainID       uint32
+	encoding      iotextypes.Encoding
+	data          []byte
+	hash          string
 }
 
 var (
 	// deterministic deployment: https://github.com/Arachnid/deterministic-deployment-proxy
 	// see example at https://etherscan.io/tx/0xeddf9e61fb9d8f5111840daef55e5fde0041f5702856532cdbb5a02998033d26
 	deterministicDeploymentTx = "f8a58085174876e800830186a08080b853604580600e600039806000f350fe7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf31ba02222222222222222222222222222222222222222222222222222222222222222a02222222222222222222222222222222222222222222222222222222222222222"
+	deterministicDeployer     = "0x3fab184622dc19b6109349b94811493bf2a45362"
 
 	// example access list tx, created with a random chainID value = 0x7a69 = 31337
 	accessListTx = "01f8a0827a690184ee6b280082520894a0ee7a142d267c1f36714e4a8f75612f20a797206480f838f794a0ee7a142d267c1f36714e4a8f75612f20a79720e1a0000000000000000000000000000000000000000000000000000000000000000001a0eb211dfd353d76d43ea31a130ff36ac4eb7b379eae4d49fa2376741daf32f9ffa07ab673241d75e103f81ddd4aa34dd6849faf2f0f593eebe61a68fed74490a348"
 
 	rlpTests = []rlpTest{
 		{
-			"transfer",
-			"",
-			0,
-			21000,
-			"1000000000000",
-			"100000000000000000",
+			"transfer", 0, 21000, "1000000000000", "100000000000000000",
 			"0x3141df3f2e4415533bb6d6be2A351B2db9ee84EF",
 			_evmNetworkID,
 			iotextypes.Encoding_ETHEREUM_EIP155,
 			nil,
 			"13d18bcf4ba5bfad081943dce8f971df64bf7abe64cf194d2d29f57d317647af",
-			"04b2113bcb7b4aefed1ee215aa8ae6818127ed7091c8d59c2dff9f053d097ec94b93be9e7399e8ee73105a0b53454cb4eb0943220b2357d0fdbd5fbffc687831ad",
-			"053b09e98ede40997546e8bb812cd838f18bb146",
 		},
 		{
-			"execution",
-			"f8ab0d85e8d4a5100082520894ac7ac39de679b19aae042c0ce19facb86e0a411780b844a9059cbb0000000000000000000000003141df3f2e4415533bb6d6be2a351b2db9ee84ef000000000000000000000000000000000000000000000000000000003b9aca008224c5a0fac4e25db03c99fec618b74a962d322a334234696eb62c7e5b9889132ff4f4d7a02c88e451572ca36b6f690ce23ff9d6695dd71e888521fa706a8fc8c279099a61",
-			13,
-			21000,
-			"1000000000000",
-			"0",
+			"execution", 13, 21000, "1000000000000", "0",
 			"0xAC7AC39De679b19AaE042c0cE19fAcB86e0A4117",
 			_evmNetworkID,
 			iotextypes.Encoding_ETHEREUM_EIP155,
 			MustNoErrorV(hex.DecodeString("a9059cbb0000000000000000000000003141df3f2e4415533bb6d6be2a351b2db9ee84ef000000000000000000000000000000000000000000000000000000003b9aca00")),
-			"7467dd6ccd4f3d7b6dc0002b26a45ad0b75a1793da4e3557cf6ff2582cbe25c9",
-			"041ba784140be115e8fa8698933e9318558a895c75c7943100f0677e4d84ff2763ff68720a0d22c12d093a2d692d1e8292c3b7672fccf3b3db46a6e0bdad93be17",
-			"87eea07540789af85b64947aea21a3f00400b597",
```

Comment:
the pubkey and address of signer, now become common values, also remove from test